service/init.go

```diff
@@ -329,10 +329,7 @@ func (amf *AMF) Start() {
 
 	addr := fmt.Sprintf("%s:%d", self.BindingIPv4, self.SBIPort)
 
-	ngapHandler := ngap_service.NGAPHandler{
-		HandleMessage:      ngap.Dispatch,
-		HandleNotification: ngap.HandleSCTPNotification,
-	}
+	ngapHandler := ngap_service.NGAPHandler(ngap.Dispatch)
```

Comment:
Is it like new sctp module does not support notification? What happens in client (i.e. gNodeB) crashes/restarts